2022/40xxx/CVE-2022-40626.json

```diff
@@ -3,16 +3,118 @@
     "data_format": "MITRE",
```

Comment:
'Zabbix' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'

Reply:
hi guys,
I have tried multiple times different values for "ASSIGNER" like "security@zabbix.com", "cve@mitre.org" and "Zabbix". Could you please comment on what should be added to this parameter?

Reply:
Hello @zabbixsecurity,

We transferred custodianship of this record from MITRE CNA-LR to Zabbix CNA. In order to get this unblocked please change the assigner value from "Zabbix" to "security@zabbix.com".

Thanks,
Anthony
CVE Content Team